packages/flame/lib/src/game/game_render_box.dart

```diff
@@ -19,9 +19,6 @@ class GameRenderBox extends RenderBox with WidgetsBindingObserver {
     WidgetsBinding.instance!.addTimingsCallback(game.onTimingsCallback);
   }
 
-  @override
-  bool get sizedByParent => true;
```

Comment:
A Flame game should fill all the size from the parent right?

Reply:
That us why receives the biggest size given the constraints of

Reply:
What is the difference in doing this to keeping `sizedByParent` and do the resizing in `performResize`?

Reply:
When size by parent is true, it will take into consideration the outcome of "computeDryLayout" as the size of this render object, this is done when the parent render object is ready to be layouted.

Reply:
Isn't that what we want?

Reply:
Nope, since we simplified to a single render object (removed the constrained render object in the previous PR), this render object now expands itself.

Reply:
Alright, so the sizedByParent is related to the parent render object, not the "real" parent, so everything will be the same.